txmongo/protocol.py

```diff
@@ -381,6 +381,11 @@ def handle_REPLY(self, request):
                 df.errback(err)
                 if fail_conn:
                     self.transport.loseConnection()
+            elif request.response_flags & REPLY_CURSOR_NOT_FOUND:
```

Comment:
Is it possible to simulate cover this in unit tests?
If it is hard to achieve with real MongoDB, is it possible to monkeypatch it with `mock.patch`?